docs/config.toml

```diff
@@ -2,74 +2,563 @@ baseurl = "https://docs.helm.sh/"
 languageCode = "en-us"
 title = "Helm Docs"
 theme = "helmdocs"
+
+contentDir = "source/docs"
 publishdir = "app"
 
 [params]
     title = "Helm Docs"
     author = "Ronan Flynn-Curran"
     description = "Documentation for Helm - The Kubernetes Package Manager."
 
-canonifyURLs = "false"
-relativeURLs = "true"
+canonifyURLs = "true"
```

Comment:
curious about the change to flip this and the `relativeURLs` values from previous?